config/locales/fr.yml

```diff
@@ -150,24 +154,24 @@ fr:
       web: Web
     action_logs:
       actions:
-        assigned_to_self_report: "%{name} s'est assigné le signalement de %{target} à eux-même"
-        change_email_user: "%{name} a modifié l'adresse de courriel de l'utilisateur %{target}"
-        confirm_user: "%{name} adresse courriel confirmée de l'utilisateur %{target}"
-        create_custom_emoji: "%{name} a importé de nouveaux emoji %{target}"
+        assigned_to_self_report: "%{name} s’est assigné le signalement de %{target} à eux-même"
```

Comment:
assigné⋅e

Reply:
*s’est [...] lui-même* or *se sont [...] eux-même*

Reply:
«s’est assigné·e» is enough, so:
`"%{name} s’est assigné·e le signalement de %{target}"`